can-define.js

```diff
@@ -281,7 +283,7 @@ make = {
 				if (newVal !== current) {
 					setData.call(this, newVal);
 
-					canEvent.dispatch.call(this, {
+					canEvent.trigger.call(this, {
```

Comment:
I think we should make this `.emit` eventually to comply with event-emitter like apis.